engine/src/qlcfixturemode.cpp

```diff
@@ -410,11 +453,22 @@ bool QLCFixtureMode::saveXML(QXmlStreamWriter *doc)
 
     /* Channels */
     QVectorIterator <QLCChannel*> it(m_channels);
-    while (it.hasNext() == true)
+    while(it.hasNext() == true)
     {
+        QLCChannel* channel = it.next();
+
         doc->writeStartElement(KXMLQLCFixtureModeChannel);
         doc->writeAttribute(KXMLQLCFixtureModeChannelNumber, QString::number(i++));
-        doc->writeCharacters(it.next()->name());
+
+        if(m_actsOnChannelsList.contains(channel))
+        {
+            QLCChannel *ChannelActsOn = m_actsOnChannelsList.value(channel);
+            if(ChannelActsOn != nullptr){
```

Comment:
Use NULL. It's backward compatible with older GCCs

Reply:
You've got 859 of nullptr in your code...